flask_dance/consumer/oauth1.py

```diff
@@ -209,7 +216,7 @@ def authorized(self):
             message = err.args[0]
             response = getattr(err, "response", None)
             log.warning("OAuth 1 access token error: %s", message)
-            oauth_error.send(self, message=message, response=response)
+            oauth_error.send(self, message=message)
```

Comment:
Please revert this change. _This_ error _will_ have a `response` property, and it should be passed along.